shim-logs.c

```diff
@@ -93,8 +95,12 @@ int main() {
     while(1) {
         int n = epoll_wait(evfd, events, MAXEVENTS, -1);
 
-        if(n < 0)
+        if(n < 0) {
+            if(errno == EINTR)
+                continue;
```

Comment:
Can this cause a `busy loop` if epoll_wait immediately returning errors each time it's called. may be it's better to introduce some latency before trying again.

Reply:
`EINTR` means syscall were interrupted by a signal. This is just to avoid to exit because of a signal which was not intended to stop the whole process.